napari/_qt/widgets/qt_welcome.py

```diff
@@ -0,0 +1,181 @@
+        )
+
+        # Widget setup
+        self.setAutoFillBackground(True)
+        self.setAcceptDrops(True)
+        self._image.setAlignment(Qt.AlignCenter)
+        self._label.setAlignment(Qt.AlignCenter)
+
+        # Layout
+        text_layout = QVBoxLayout()
+        text_layout.addWidget(self._label)
+
+        # TODO: Use action manager for shortcut query and handling
+        shortcut_layout = QFormLayout()
+        sc = QKeySequence('Ctrl+O').toString(QKeySequence.NativeText)
+        shortcut_layout.addRow(
+            QtShortcutLabel(sc),
+            QtShortcutLabel(trans._("open image(s)")),
+        )
+        sc = QKeySequence('Ctrl+Alt+/').toString(QKeySequence.NativeText)
+        shortcut_layout.addRow(
+            QtShortcutLabel(sc),
+            QtShortcutLabel(trans._("show all key bindings")),
+        )
+        shortcut_layout.setSpacing(0)
+
+        layout = QVBoxLayout()
+        layout.addStretch()
+        layout.setSpacing(30)
+        layout.addWidget(self._image)
+        layout.addLayout(text_layout)
+        layout.addLayout(shortcut_layout)
+        layout.addStretch()
+
+        self.setLayout(layout)
+
+    def paintEvent(self, event):
+        """Override Qt method.
+
+        Parameters
+        ----------
+        event : qtpy.QtCore.QEvent
+            Event from the Qt context.
+        """
+        option = QStyleOption()
+        option.initFrom(self)
+        p = QPainter(self)
+        self.style().drawPrimitive(QStyle.PE_Widget, option, p, self)
+
+    def _update_property(self, prop, value):
+        """Update properties of widget to update style.
+
+        Parameters
+        ----------
+        prop : str
+            Property name to update.
+        value : bool
+            Property value to update.
+        """
+        self.setProperty(prop, value)
+        self.style().unpolish(self)
+        self.style().polish(self)
+
+    def dragEnterEvent(self, event):
+        """Override Qt method.
+
+        Provide style updates on event.
+
+        Parameters
+        ----------
+        event : qtpy.QtCore.QEvent
+            Event from the Qt context.
+        """
+        self._update_property("drag", True)
+        if event.mimeData().hasUrls():
+            event.accept()
+        else:
+            event.ignore()
+
+    def dragLeaveEvent(self, event):
+        """Override Qt method.
+
+        Provide style updates on event.
+
+        Parameters
+        ----------
+        event : qtpy.QtCore.QEvent
+            Event from the Qt context.
+        """
+        self._update_property("drag", False)
+
+    def dropEvent(self, event):
+        """Override Qt method.
+
+        Provide style updates on event and emit the drop event.
+
+        Parameters
+        ----------
+        event : qtpy.QtCore.QEvent
+            Event from the Qt context.
+        """
+        self._update_property("drag", False)
+        self.sig_dropped.emit(event)
+
+
+class QtCanvasOverlay(QWidget):
```

Comment:
see https://github.com/goanpeca/napari/pull/3 for thoughts on trimming this down.  namely, `QtCanvasOverlay` could just be a `QStackedWidget` instead of having one.  And the `sig_dropped` can be removed if you just call the `dropEvent` of QtViewer from the welcome widget.